src/haz3lweb/www/index.html

```diff
@@ -9,6 +9,26 @@
     <title>hazel</title>
     <link rel="stylesheet" href="style.css">
     <link rel="icon" type="image/svg" href = "img/hazelnut.svg">
+    <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin />
+    <link
+      rel="preload"
+      href="https://fonts.googleapis.com/css?family=Inter:300,400,500"
+      as="style"
+      type="text/css"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Material+Icons&display=swap"
+      rel="stylesheet"
+    />
+    <script type="importmap">
+      {
+          "imports": {
+              "https://unpkg.com/lit-html@latest/directives/ref.js?module": "https://unpkg.com/lit-html@2.2.6/directives/ref.js?module"
+          }
+      }
```

Comment:
The current version of lit-html is broken so this overwrites it for the ninja-keys import.

Reply:
would be good to leave this as a comment in the code